march_hardware/src/joint.cpp

```diff
@@ -131,6 +131,15 @@ double Joint::getVelocity() const
   return this->velocity_;
 }
 
+double Joint::getVoltageVelocity() const
+{
+  double R_motor = 0.05;
+  double Kv_motor = 355;
+  double rpmToRad = M_PI / 30;
+  double Ke = Kv_motor * rpmToRad;
+  return (this->imc_->getMotorVoltage() + this->imc_->getMotorCurrent() * R_motor) / Ke;
```

Comment:
Maybe add a reference or link here to where this calculation comes from?

Reply:
I have this link that explains the calculation: https://www.motioncontroltips.com/faq-whats-relationship-voltage-dc-motor-output-speed/. Should I add this one or a confluence page where I explain it?

Reply:
@Roelemans ?

Reply:
That link is fine :+1:

Reply:
That link is fine :+1: